src/resources/errors.js

```diff
@@ -1,8 +1,17 @@
+// eslint-disable-next-line max-classes-per-file
 class SendMessageBatchError extends Error {
   constructor(message) {
     super(message);
     this.name = 'SendMessageBatchError';
   }
 }
 
+class MessageCollisionError extends Error {
+  constructor(message) {
+    super(JSON.stringify(message));
```

Comment:
We need to do this for the `SendMessageBatchError` error (`JSON.stringify`).

Reply:
Why what?